k8s/wordpress/manifest/manifests.yaml.template

```diff
@@ -170,9 +176,26 @@ spec:
         - name: $APP_INSTANCE_NAME-wordpress-pvc
           mountPath: /var/www/html
           subPath: wp
+        lifecycle:
+          postStart:
+            exec:
+              command:
+              - "/bin/bash"
+              - "-euc"
+              - |
+                # https://codex.wordpress.org/Installing_WordPress#Installing_WordPress
+
+                curl --fail --retry 5 --retry-delay 5 --retry-connrefused \
+                  --data-urlencode "weblog_title=WordPress on Google Kubernetes Engine" \
+                  --data-urlencode "user_name=${WP_ADMIN_USERNAME}" \
+                  --data-urlencode "admin_email=noreply@google.com" \
```

Comment:
It should be a real email address of WordPress administrator.

Reply:
Generally I agree and I've prepared a change for this - https://github.com/GoogleCloudPlatform/click-to-deploy/commit/ab0675ea0d64bce8212e81ca981c3fc6c11c66de.

However, I'm worried about pattern for e-mail address. Which one would you like to use?

If we hard-coded the e-mail address, we don't have to care and think about such regex.

Please let me know.

Reply:
Done.